amoro-format-mixed/amoro-mixed-trino/src/test/java/org/apache/amoro/trino/mixed/TestKeyedTable.java

```diff
@@ -62,7 +62,7 @@ public void testStats() {
         .matches(
             "VALUES "
                 + "('id', NULL, NULL, 0e0, NULL, '1', '4'), "
-                + "('name$name', 548e0, NULL, 0e0, NULL, NULL, NULL), "
+                + "('name$name', 429e0, NULL, 0e0, NULL, NULL, NULL), "
```

Comment:
It is a estimation value and used by query optimize and cost estimation. So I think the change is not a issue